pkg/services/object/util/key.go

```diff
@@ -51,10 +51,13 @@ type SessionInfo struct {
 	Owner user.ID
 }
 
-// GetKey returns private key of the node.
+// GetKey fetches private key.
```

Comment:
`of the node`?

Reply:
it is not the node's key if a dynamic session token is used (`info != nil`)

Reply:
It is just that `fetches private key` is obvious from the function signature, can we clarify somehow?

Reply:
changed to `// GetKey fetches private key depending on the SessionInfo.`